onnx/defs/rnn/defs.cc

```diff
@@ -5,3 +5,114 @@
 
 using AttrType = onnx::OpSchema::AttrType;
 
+// CuDNN parameters not included yet:
+// - dropout (as we primarily target inference)
+// Description below is borrowed from CuDNN and TensorRT docs
+
+OPERATOR_SCHEMA(OptimizedRNN)
+    .NumInputs(2, 4)
+    .NumOutputs(1, 3)
+    .SetDoc(R"DOC(
+Computes a stack of several RNNs in optimized fashion. This operator corresponds
+to CuDNN functionality and serves as a thin layer on top.
```

Comment:
ONNX doesn't mandate implementation, right? So I'd reword this as, "This operator is usually implemented via CuDNN"